client/post-editor/editor-drawer/copy-post.jsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import Accordion from 'components/accordion';
+import AccordionSection from 'components/accordion/section';
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
+import PostSelector from 'my-sites/post-selector';
+
+class EditorDrawerCopyPost extends Component {
+
+	static propTypes = {
+		site: PropTypes.object.isRequired,
+		post: PropTypes.object.isRequired,
+		translate: PropTypes.func,
+	};
+
+	constructor() {
+		super( ...arguments );
+		this.state = {
+			selectedPostID: '',
+		};
+	}
+
+	setPostToCopy = post => {
+		this.setState( {
+			selectedPostID: post.ID,
+		} );
+	}
+
+	goToNewDraft = () => {
+		if ( '' !== this.state.selectedPostID ) {
+			page.redirect( `/post/${ this.props.site.slug }/${ this.state.selectedPostID }?copy=true` );
```

Comment:
Oh, great! I didn't notice `page.replace()` in page.js docs. I should have checked the actual code instead.

As for the variable name: my bad, uppercase ID is the convention we use at work, and it totally slipped here.